lib/power_action_utils.pm

```diff
@@ -81,15 +81,21 @@ sub reboot_x11 {
     my ($self) = @_;
     wait_still_screen;
     if (check_var('DESKTOP', 'gnome')) {
-        send_key_until_needlematch 'logoutdialog', 'ctrl-alt-delete', 7, 10;    # reboot
+        if (is_tumbleweed) {
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check.
@ggkioulis I would appreciate if you can look into this and create another PR for that. Otherwise you would need to look into this anyway in about 1 year when the new gnome version reaches SLE :D

Reply:
Good point, thanks Oliver! I will submit today a change where `is_tumbleweed` is not used and the changed gnome reboot approach is the default.